### DISCUSSION TITLE
how to deploy an APP NodeJS Through cPan... [May 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2119] [Discord Threads]
> 🙋‍♂️ **Paul Taylor**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### did you try this project ? https://github.com/nocodb/nocodb-seed  
#  
> 🙋‍♂️ **Paul Taylor**  
#### Ok thanks will take a look cheers  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/972681508311666758)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
